documentation/configuringDynawo/DynawoInputFiles.tex

```diff
@@ -36,23 +36,38 @@ \section{Overview}
 
 \subsection[Dynawo inputs]{\Dynawo inputs}
 
-To run a simulation, \Dynawo needs most of the time five input files:
+To run a simulation, \Dynawo needs most of the time four input files:
 \begin{itemize}
 \item A file providing the simulation settings: the jobs file (\textbf{.jobs} extension);
 \item A file providing the static description of the test grid to simulate: the iidm file (\textbf{.iidm} extension);
 \item A file providing the dynamic description of the test grid to simulate: the dynamic data file (\textbf{.dyd} extension);
 \item A file providing the needed parameters of the simulation: the parameter file (\textbf{.par} extension);
+\end{itemize}
+The iidm file can be optional for small test cases where the whole system is represented in the dyd file (see paragraph \ref{sec:Run a simulation with no iidm file}).
+Other files are mandatory. \\
+
+Others optional files can be given as inputs:
+\begin{itemize}
 \item A file providing the list of the variables to plot at the end of the simulation: the curves file (\textbf{.crv} extension);
+\item A file providing the list of the variables for which we want to observe the final value: the final state value file (\textbf{.fsv} extension);
```

Comment:
Maybe we can simplify this sentence: "A file providing the final values for a list of variables: the final state value file ..."

Reply:
Done